### ISSUE TITLE
Update of framework fails on third node with KeyError: aliases

### ISSUE THREAD

**Issue by pploegaert** [openvstorage/framework-alba-plugin#260]
```
2016-11-03 11:34:33 72000 +0100 - e187-3 - 124762/140036233402176 - lib/update - 118 - DEBUG -   10.100.187.31   - Service arakoon-ovsdb started
2016-11-03 11:34:33 72100 +0100 - e187-3 - 124762/140036233402176 - lib/update - 119 - INFO - 10.100.187.31  : Started service arakoon-ovsdb
2016-11-03 11:34:33 72100 +0100 - e187-3 - 124762/140036233402176 - lib/update - 120 - INFO - 10.100.187.33  : Started model migration
2016-11-03 11:34:37 51800 +0100 - e187-3 - 124762/140036233402176 - lib/update - 121 - ERROR - 10.100.187.33  : An unexpected error occurred: 'aliases'

========= Remote Traceback (1) =========
Traceback (most recent call last):
  File "/tmp/tmp.ODi6bcqQN8/rpyc/core/protocol.py", line 305, in _dispatch_request
    res = self._HANDLERS[handler](self, *args)
  File "/tmp/tmp.ODi6bcqQN8/rpyc/core/protocol.py", line 535, in _handle_call
    return self._local_objects[oid](*args, **dict(kwargs))
  File "/opt/OpenvStorage/ovs/dal/helpers.py", line 342, in migrate
    new_version = execute(method, base_version, 0)
  File "/opt/OpenvStorage/ovs/dal/helpers.py", line 317, in execute
    version = function(start)
  File "/opt/OpenvStorage/ovs/dal/migration/albamigrator.py", line 114, in migrate
    if '/dev/disk/by-id/{0}'.format(alba_disk._data['name']) in disk_info['aliases']:
KeyError: 'aliases'
```

**Comment by khenderick** [comment]
This ticket will only resolve this particular issue by removing some of the migration code. Continued in openvstorage/framework#1118. As long as that ticket is not resolved, the migration will not be complete (but it is not critical for the system to work)

**Comment by khenderick** [comment]
Fixed by #261, packaged in openvstorage-backend-1.7.4-rev.803.b142af7

**Comment by pploegaert** [comment]
PASSED - manual validation
